bundle/Form/FieldTypeHandler/EnhancedSelection.php

```diff
@@ -74,6 +85,10 @@ protected function buildFieldForm(
         $optionsValues = $fieldSettings['options'];
 
         $options['expanded'] = false;
+        if ($this->configResolver->hasParameter('form.handler.expanded', 'netgen.enhanced_selection')) {
```

Comment:
There's no need for `hasParameter` check. The parameter is always defined since, together with my other comment, it exists for the `default` scope at least.